### ISSUE TITLE
change deep sort to sort

### ISSUE THREAD

**Issue by LeeRock** [nwojke/deep_sort#129]
dear @nwojke,is there any  way to change your code to sort(sort with with matching cascade) ?I mean just disable all code using appearance features distance.Could you show me some lines in your code which must be changed?

**Comment by antocapp** [comment]
maybe you are looking for this https://github.com/abewley/sort ?

**Comment by LeeRock** [comment]
> maybe you are looking for this https://github.com/abewley/sort ?
No.I had checked that repo.I just want to know how to modify a bit code in deep sort to make 
 it work like sort .....

**Comment by LeeRock** [comment]
> maybe you are looking for this https://github.com/abewley/sort ?

No.I had checked that repo.I just want to know how to modify a bit code in deep sort to make
it work like sort ....

**Comment by PoonamZ** [comment]
@LeeRock Do you have any idea about how to modify sort to add only id's over the each object ?

**Comment by nwojke** [comment]
Hi LeeRock, sorry for the late reply. It is fairly straight-forward to make the necessary adaptions to end up with a code base that runs SORT with matching cascade. Most importantly, you will need to change [this line](https://github.com/nwojke/deep_sort/blob/master/deep_sort/tracker.py#L112) to use ``iou_matching.iou_cost`` instead of ``gated_metric``.

Another approachw ould be to remove ``deep_sort.track.Track.features`` and then continue to remove any code fragment that causes an error on execution. That way you can be sure no appearance information are used.

Hope it helps.

**Comment by ruksin** [comment]
> Hi LeeRock, sorry for the late reply. It is fairly straight-forward to make the necessary adaptions to end up with a code base that runs SORT with matching cascade. Most importantly, you will need to change [this line](https://github.com/nwojke/deep_sort/blob/master/deep_sort/tracker.py#L112) to use `iou_matching.iou_cost` instead of `gated_metric`.
> 
> Another approach would be to remove `deep_sort.track.Track.features` and then continue to remove any code fragment that causes an error on execution. That way you can be sure no appearance information are used.
> 
> Hope it helps.

how can we only use appearance features and not include iou matching?

**Comment by amunamun07** [comment]
Hey @nwojke , I wanted to use just the SORT and nothing else using this same repo. How will i be able to achieve that?
